app/Http/Controllers/LateFeeController.php

```diff
@@ -32,7 +32,12 @@ public function update(LateFee $lateFee, Request $request)
     {
         // dd($request['amount_paid']);
         $today = Carbon::now()->toDateString();
-        $lateFee->date_paid = $today;
+        if($request->date_paid != null){
+            $lateFee->date_paid = $request->date_paid;
```

Comment:
why not a 1 liner? `$lateFee->date_paid = $request->date_paid ?? $today`

Reply:
done. Thanks